File: Lec -1 C Programming/agree.c
#include<stdio.h>
#include <cs50.c>
int main (void)
{
  char c = get_char("Do you agree ? ");
  
  if (c == 'y' || c == 'Y')
  {
    printf("You agreed!\n");
  }
  else if( c == 'n' || c == 'N')
  {
    printf("You disagreed!\n");
  }
  return 0;


}

File: Lec - 6 Python/agree.py
import cs50

c = cs50.get_string ("Do you agree? ")

#if c == 'Y' or c == 'y' :
if c.lower() in ["y","yes"]:
    print("Agree")
    
#elif c == 'N' or c == 'N':
elif c.lower() in ["n","no"]:
    print("Disagree")

